### ISSUE TITLE
fix rails new with spockets for windows vagrant

### ISSUE THREAD

**Issue by kookster** [railsbridge-boston/railsbridge-boston#119]
We're getting this error:

http://stackoverflow.com/questions/19124367/rails-app-on-percise32-vagrant-box-assets-get-text-file-busy-error-errnoe

looks like sass may be the culprit, see if we can just use a different version.

if not, look at alternative ways to get assets (compile once, app template, download)

**Comment by thegreatape** [comment]
I tried to reproduce the "text file busy" issue on my 64-bit Windows 7 machine, with no success. The only way the setup deviated from the install instructions was that I ran "rails new" without the no-sprockets flag and I had to install nodejs with apt-get for the asset pipeline. 

I tried with sass 3.2.10, 3.2.11, and 3.2.12 and all worked fine. Thoughts on anything I might have missed?

Having people next time who run into this issue try downgrading sass to 3.2.10 per that stackoverflow is a decent idea, but until we can reproduce the issue, it's just an educated shot in the dark.

**Comment by danchoi** [comment]
@slothbear and I tried to debug this issue the day before the Friday installfest, but we just gave up and turned to the `--skip-sprockets` solution, which is not ideal for obvious reasons.

This "text file busy" error is erratic, but @slothbear and I both confirmed it on 2 different Windows machines, one Windows 7 and one Windows 8.

I also tried dumping `nodejs` as the asset pipeline compliation engine and switching to `execjs` and `therubyracer`, but the error still kept happening.
